addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/options.html

```diff
@@ -1,33 +1,55 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
-<HTML>
-<HEAD>
-<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
-<TITLE>
-Options Fuzz screen
-</TITLE>
-</HEAD>
-<BODY>
-<H1>Options Fuzzer screen</H1>
-<p>
-This screen allows you to configure the <a href="concepts.html">fuzzing</a> options:
-
-<H3>Default category</H3>
-The category that will be initially selected shown when the <a href="dialogue.html">Fuzz dialog</a> is displayed. 
-
-<H3>Concurrent scanning threads per host</H3>
-The number of threads the scanner will use per host.<br>
-Increasing the number of threads will speed up the scan but may put extra strain on the computer ZAP is running on and the target host.
-
-<H3>Add custom Fuzz file</H3>
-Allows you to add your own files to be used when fuzzing.<br>
-These should be text files with one attack per line.<br>
-Files are added to the 'fuzzers' directory underneath the ZAP home directory.
-
-<H2>See also</H2>
-<table>
-<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;</td><td><a href="concepts.html">Fuzzer concepts</a></td></tr>
-</table>
-
-
-</BODY>
-</HTML>
\ No newline at end of file
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Fuzzer Options Screen</title>
+  </head>
+  <body>
+    <h1>Fuzzer Options Screen</h1>
```

Comment:
This is one item that remains contrary to the Title Caps issue. We need to establish a standard (and update ALL the docs) to accommodate how "screen", "tab", "dialog" etc should be noted in titles. In general most are currently lower case with no punctuation.

In the past I've suggested we use either brackets `(Screen)` or hyphen `- Screen` but we never landed on an agreed format, so for now it's probably best to leave this as-is.